app/util.js

```diff
@@ -58,5 +58,12 @@ utils.addPrompt = function addPrompt(promptContent, promptsList) {
     }
 }
 
+utils.validatePackage = function(packageName) {
+    var regex = /^[a-z][a-z0-9_]*(\.[a-z0-9_]+)+[0-9a-z_]$/;
```

Comment:
some use cases not covered:
- not detecting invalid package name when containing a Java keyword, for instance _a.name.with.package_
- report false positive for greek characters and accent, for instance _à.test_
- report false positive for upper case characters (even if bad practice) for instance _a.A_

not sure what is best, forcing more restrictive package names or let users do what he wants and maybe fall in trap.
I think we might need to change the error message to explain that it is not "an invalid package name" but a "not supported package name" by the tool and point to the regex

Reply:
I don't think we can really support every use case here. I can add some checking for java keywords, but I think some common sense needs to apply. I will add a new task to add some documentation around these prompts to hopefully guide the user a bit more constructively.

Reply:
i think that the error message should at least be modified to avoid saying false statement.
When saying: "invalid package name" although it is a valid one, the software has a bug
When "saying" Unsupported package name by this tool", the software has a limited scope of functionality.